guides/v2.1/cloud/project/new-relic.md

```diff
@@ -18,13 +17,47 @@ New Relic APM provides the following features to Magento:
 -  **App Map**—View all application dependencies within your Magento site, extensions, and external services.
 -  **Apdex scores**—Evaluate performance and create alerts that identify issues and notify you when they occur, such as site performance affected by a flash sale or web event.
 
-Your {{site.data.var.ece}} account includes the agent software for New Relic; you do not need to purchase or install the New Relic extension (different than the APM service). You receive your credentials and license information from Magento during launch and can access your license key in the _Project Web Interface_ by clicking **View Details** for your project. The **NewRelic Service** section includes your Account Number, License Key, and other access keys.
+By default, the {site.data.var.ece}} Pro plan includes New Relic on the Staging and Production environments. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation. For {{site.data.var.ece}} Pro plan customers also need to submit a support ticket to use New Relic on an Integration branch. 
 
-By default, the Pro Staging and Production environments include New Relic. If it is not installed, submit a [support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html) to request installation.
 
-## Add New Relic extension to your project
+{:.bs-callout .bs-callout-info}
+Your Magento Commerce Cloud account includes the software for the New Relic APM service along with a license key. You do not need to purchase or install any additional software.
 
-You must list the New relic extension in the `.magento.app.yaml` file:
+## New Relic account credentials
+
+When Magento provisions your {{site.data.var.ece}} project, the Project Owner should receive an email from New Relic with the credentials and instructions for accessing the account. If you did not receive the email, you can use the Project Owner email address to [reset the password](https://rpm.newrelic.com/forgot_password){:target="_blank"}.
+
+If you cannot access the New Relic account, submit a [Magento support ticket]({{ page.baseurl }}/cloud/trouble/trouble.html).
+
+
+## Add account users
+
+{{site.data.var.ece}} Project Owners and Admin users can add and remove users from the New Relic account.
+
+We recommend adding at least one user to your New Relic account to manage all access, integrations, and usage of the tool. We also recommend assigning the Admin role to another user on the account so they can manage all New Relic access and integrations.
```

Comment:
Are you trying to say "we recommend at least two users for your New Relic account that can manage all access, integrations and usage of the tool"?